storage/src/cache/cachedfile.rs

```diff
@@ -352,10 +352,21 @@ impl BlobObject for FileCacheEntry {
             return Ok(0);
         }
 
+        let mut size = chunks[0].compress_size();
         for idx in 1..chunks.len() {
             if chunks[idx - 1].id() + 1 != chunks[idx].id() {
                 return Err(einval!("chunks for fetch_chunks() must be continuous"));
             }
+            size += chunks[idx].compress_size();
```

Comment:
Using the insane check is fine. Can we just use `BlobIoRange::blob_size` in case the insane check is removed in the future

Reply:
Good point，fixed